sbin/submitty_daemon_jobs/submitty_jobs/jobs.py

```diff
@@ -52,8 +52,7 @@ def validate_job_details(self):
     def run_job(self):
         pass
 
-    @abstractmethod
-    def cleanup_job(self):
+    def cleanup_job(self):  # noqa: B027
         pass
```

Comment:
Does changing this to a bare `return` also fix the `B027` error? Perhaps not as "pythonic" for an empty function, but :shrug:

Reply:
There were only a limited number of locations where this function needed to be implemented so I have just gone ahead and implemented it in those locations.